frontend-react/src/styles/_uswds_overrides.scss

```diff
@@ -19,6 +19,8 @@ i.e.
 @include u-padding-right('05');
 ----------------------------------------
 */
+$bg-primary: #005ea2;
```

Comment:
Hmmm I realize these hardcoded values already existed beforehand, but the hex code is already available from the USWDS CSS as `primary`.  Like we could just use `color('primary')` instead of creating a new variable 🤷